### PR TITLE
Provide all possible keywords in xml doc comment `langword` tag

### PR DESCRIPTION
**Pull request by DoctorKrolic**
Fixes: https://github.com/dotnet/roslyn/issues/72258

### CODE REVIEW

File: src/EditorFeatures/CSharpTest/Completion/CompletionProviders/XmlDocumentationCommentCompletionProviderTests.cs [hunk 1]
```diff
@@ -882,7 +882,21 @@ static void Goo()
                     {
                     }
                 }
-                """, "null", "true", "false", "await");
+                """;
+
+            foreach (var keywordKind in SyntaxFacts.GetKeywordKinds())
+            {
+                var keywordText = SyntaxFacts.GetText(keywordKind);
```
Comment (DoctorKrolic) [inline src/EditorFeatures/CSharpTest/Completion/CompletionProviders/XmlDocumentationCommentCompletionProviderTests.cs:889]: This is reusing the same logic as the actual product code, which isn't great for a test at all, but I cannot think of a better solution here. Another option is to manually list all keywords we need to check for, but this is an unmanageble nightmare. We sort of want keyword list to be tight to `SyntaxFacts.GetKeywordKinds`, so as new keywords are added to the language we immediately test that they are in the list. Same thing on VB side

**Review by CyrusNajmabadi — APPROVED** [review]

**Comment by CyrusNajmabadi** [comment]
test failure seems legit.

**Comment by DoctorKrolic** [comment]
@CyrusNajmabadi PTAL

**Review by CyrusNajmabadi — APPROVED** [review]

**Comment by CyrusNajmabadi** [comment]
Thanks!

**Pull request merged**